### ISSUE TITLE
Bad string comparison in DashboardTableNode class

### ISSUE THREAD

**Issue by rbeckman-nextgen** [nextgenhealthcare/connect#3138]
Fix string comparison in DashboardTableNode.setStatus to use String.equals instead of == operator.

Imported Issue. Original Details:
Jira Issue Key: MIRTH-3227
Reporter: brentm
Created: 2014-04-11T14:57:11.000-0700

**Comment by rbeckman-nextgen** [comment]
Fixed it to use String.equals().

Apparently these strings always did reference the same string object, so this never posed a problem.

A simple change in the code such as this.channelId = new String(channelId) in the constructor would cause it to break.

I tested creating and deploying channels with the new String(channelId) change and the String.equals() change to verify that it works correctly.


Imported Comment. Original Details:
Author: brentm
Created: 2014-04-11T15:11:46.000-0700
